do.sh

```diff
@@ -281,6 +281,19 @@ e2e_lint() {
   fi
 }
 
+e2e_build_docs() {
+  e2e_build_templates
+  # Building JS-Dossier requires Facebook Buck (http://buckbuild.com/)
+  # It's not part of a default install.
```

Comment:
I assume so. Similar to `/.do.sh lint` build_docs requires additional dependencies that might require root access to install, so I don't install them by default. Also because it's a task done by minority of people using e2e repo.